packages/terra-application-navigation/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "terra-application-navigation",
+  "main": "lib/ApplicationNavigation.js",
+  "version": "0.0.0",
+  "description": "The Terra Application Navigation is a responsive, themeable layout for building applications. ApplicationNavigation is comprised of selectable navigation and utility elements with selection callbacks.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-application-navigation",
+    "ApplicationNavigation",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "devDependencies": {
+    "terra-application": "^0.0.0"
+  },
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": "^2.4.0",
+    "terra-application": "^0.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "focus-trap-react": "^4.0.1",
+    "keycode-js": "^1.0.4",
+    "lodash.debounce": "^4.0.8",
+    "react-onclickoutside": "^6.7.1",
+    "resize-observer-polyfill": "^1.5.1",
+    "terra-action-footer": "^2.5.0",
+    "terra-avatar": "^2.7.0",
+    "terra-button": "^3.7.0",
+    "terra-doc-template": "^2.9.0",
+    "terra-popup": "^5.2.0",
+    "terra-icon": "^3.3.0",
+    "terra-overlay": "3.13.0",
```

Comment:
unlock this guy

Reply:
Updated.

Reply:
Had to revert this. There is a conflict with wicg-inert between terra-overlay and the modal.

Reply:
What's the conflict?

Reply:
I looks like both the terra-abstract-modal and terra-overlay are attempting to define inert at run time. There is an error:
TypeError: Cannot redefine property: inert

Reply:
```
Uncaught TypeError: Cannot redefine property: inert
    at Function.defineProperty (<anonymous>)
    at eval (webpack-internal:///./packages/terra-application-navigation/node_modules/terra-overlay/node_modules/wicg-inert/dist/inert.js:791)
    at eval (webpack-internal:///./packages/terra-application-navigation/node_modules/terra-overlay/node_modules/wicg-inert/dist/inert.js:2)
    at eval (webpack-internal:///./packages/terra-application-navigation/node_modules/terra-overlay/node_modules/wicg-inert/dist/inert.js:4)
    at Object../packages/terra-application-navigation/node_modules/terra-overlay/node_modules/wicg-inert/dist/inert.js (terra-dev-site.js:20753)
    at __webpack_require__ (terra-dev-site.js:64)
    at eval (webpack-internal:///./packages/terra-application-navigation/node_modules/terra-overlay/lib/Overlay.js:60)
    at Object../packages/terra-application-navigation/node_modules/terra-overlay/lib/Overlay.js (terra-dev-site.js:20474)
    at __webpack_require__ (terra-dev-site.js:64)
    at eval (webpack-internal:///./packages/terra-application-navigation/src/ApplicationNavigation.jsx:10)
```

Reply:
I'm wondering if we should treat inert like babel-polyfill and include it at webpack time.

Reply:
I've got a PR out that should address this: https://github.com/cerner/terra-core/pull/2441/files

Reply:
We've released `terra-overlay@3.14.0` that should work without throwing the `Cannot redefine property: inert` error.

Reply:
Looks like this has been unlocked in this commit: https://github.com/cerner/terra-framework/pull/695/commits/de7371595d50c9e9267e0aa690cdf57d39a290e0